src/test/java/org/fundacionjala/trello/runner/Runner.java

```diff
@@ -0,0 +1,38 @@
+package org.fundacionjala.trello.runner;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.DataProvider;
+
+/**
+ * Cucumber TestNG runner class.
+ */
+@CucumberOptions(
+        plugin = {"pretty"},
+        features = {"src/test/resources/features"},
+        glue = {"org.fundacionjala.trello"}
+)
+public final class Runner extends AbstractTestNGCucumberTests {
+    @Override
+    @DataProvider
```

Comment:
Please consider to allow parallel execution of BDD Tests, adding: "@DataProvider(parallel = true)"

Reply:
Thanks 
I solved the observation